x-pack/platform/packages/shared/kbn-ai-assistant/src/utils/get_model_options_for_inference_endpoints.test.ts

```diff
@@ -14,27 +14,97 @@ import {
   ModelOptionsData,
 } from './get_model_options_for_inference_endpoints';
 import type { InferenceAPIConfigResponse } from '@kbn/ml-trained-models-utils';
+import {
+  DEFAULT_ELSER_INFERENCE_ID,
```

Comment:
Perhaps we should rename this in a way so it's easy to distinguish it from the EIS endpoint

```suggestion
  ELSER_ON_ML_NODE_INFERENCE_ID,
```

Reply:
Thanks @sorenlouv 
I updated it